hnspolicy.go

```diff
@@ -9,18 +9,18 @@ type PolicyType = hns.PolicyType
 
 // RequestType const
 const (
-	Nat                  = hns.Nat
-	ACL                  = hns.ACL
-	PA                   = hns.PA
+	NAT                  = hns.NAT
```

Comment:
Wont these all be breaking changes?

Reply:
I changed the capitalization to better match the V2 Schema. If that results in a breaking change, maybe this is the time to pay those costs.

Reply:
@jhowardmsft - Is it likely just Docker we will have to update here or do we have any idea how many consumers this might break?